### PR TITLE
Fix some broken links

### LINKED ISSUE #17760: Broken link
> For more information, see the Tiered compilation guide.

Should link to: https://github.com/dotnet/runtime/blob/master/docs/design/features/tiered-compilation.md

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 70b4e642-6b61-128c-3eba-2f0ca8ff440c
* Version Independent ID: 4fe316ee-08aa-2989-af97-2bfc0f698f01
* Content: [Compilation config settings - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/compilation#feedback)
* Content Source: [docs/core/run-time-config/compilation.md](https://github.com/dotnet/docs/blob/master/docs/core/run-time-config/compilation.md)
* Product: **dotnet-core**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by gewarren**
(Per the OPS portal report.)

Fixes #17760.

Contributes to #17649.

### CODE REVIEW

File: docs/azure/migration/choose.md [hunk 1]
```diff
@@ -25,19 +25,19 @@ The following table shows several comparisons and recommendations to help you ch
...
+|Cons             |It is IaaS. Maintenance is costly. You have to manage the VM's infrastructure about networking, load-balancer, scale-out, IIS management, and so on. |<ul><li>Not all apps are [supported](https://appmigration.microsoft.com/assessment)</li><li>Some apps might need to be refactored and even slightly rearchitected, so they support Azure App Service.</li></ul> |<ul><li>Docker's skills learning curve</li><li>Some code and app configuration settings changes</li></ul>|
+|Requirements |Windows Server VM with the same requirements than the app for on-premises | Azure App Service requirements specified in [Readiness checks](https://github.com/Azure/App-Service-Migration-Assistant/wiki/Readiness-Checks). |<ul><li>Windows Server 2016 with Containers - Azure VM<br />or</li><li>[Azure Container Service (AKS)](https://azure.microsoft.com/services/container-service/) (That is Kubernetes orchestrator)<br />or<li>[Azure Service Fabric](https://azure.microsoft.com/services/service-fabric/) orchestrator</li></ul> |
```
Comment (gewarren) [inline docs/azure/migration/choose.md:31]: @CamSoper I could not find a Windows Server 2016 with Containers app on the Azure marketplace so I just removed the link. LMK if there's a link I can put there instead.

**Review by BillWagner — APPROVED** [review]
This LGM @gewarren 

Once you resolve the question with @CamSoper you can :shipit:

File: docs/azure/migration/choose.md (hunk 1 above)
Comment (CamSoper) [reply docs/azure/migration/choose.md:31]: Looks like the ARM template was removed in favor of letting Docker make their own. The best fit is:

[Docker Engine on Windows](https://azuremarketplace.microsoft.com/marketplace/apps/cloud-infrastructure-services.docker-windows-2019)

**Review by CamSoper — APPROVED** [review]
The link you want for that question is in the review comments. Other than that, LGTM. :shipit:

File: docs/azure/migration/choose.md (hunk 1 above)
Comment (gewarren) [reply docs/azure/migration/choose.md]: Thanks Cam!

**Pull request merged**